Sources/Logging/Strings/OfferingStrings.swift

```diff
@@ -155,6 +156,12 @@ extension OfferingStrings: LogMessage {
 
         case let .overriding_package(old, new):
             return "Package: \(old) already exists, overwriting with: \(new)"
+
+        case .known_issue_ios_18_4_simulator:
+            return "None of the products registered in the RevenueCat dashboard could be fetched from App Store " +
+            "Connect (or the StoreKit Configuration file if one is being used)." +
+            "\nThis issue is widely reported by iOS 18.4 simulator users. Try using a different simulator version." +
```

Comment:
```suggestion
            "\nThis issue is widely reported by iOS 18.4 simulator users. Try using a different iOS version with your simulator." +
```

Nit: Technically the issue depends on the OS version in the simulator, not the simulator version. For example, the bug presents in Xcode 16.3 simulators running iOS 18.4, but not in Xcode 16.3 simulators running iOS 18.3

Reply:
Good point, resolved in 7f09f8520a63626f2238f9759f3291fc2c036656